api/proto/config/devmodel.proto

```diff
@@ -89,27 +89,36 @@ message SystemAdapter {
   // ( physical interface), this should point to PhyLabel of the
   // physicalIO.
   string lowerLayerName = 8;
+
+  // fallback_priority
+  //  0 is the highest priority; 255 the lowest
+  //  Lower priority interfaces are used only when NONE of the higher
+  //  priority interfaces are up.
+  //  The largest numeric value between this and the fallback_priority in
+  //  PhyIOUsagePolicy is what is used.
+  uint32 fallback_priority = 9;
 }
 
-// Given additional details for EVE softwar to how to treat this
+// Given additional details for EVE software to how to treat this
 // interface. Example policies could be limit use of LTE interface
 // or only use Eth1 only if Eth0 is not available etc
-// XXX Note that this is the static information from the model.
-// Current configuration is in systemAdapter
+// Note that this is the static information from the model, and
+// systemAdapter information is combined with this so that the largest
+// numeric fallback_priority is used.
 message PhyIOUsagePolicy {
-  // DEPRECATED - Used only when one other normal uplinks are down
+  // DEPRECATED by fallback_priority - Used only when one other normal uplinks are down
```

Comment:
It would be best to explicitly document how the boolean stacks up with the uint32 value (sort of the way you did in the PR description @eriknordmark)

Reply:
Good point @rvs. I'll find a place (docs/CONFIG.md is the best match I can find)

Reply:
@rvs Documentation has been added (and @ppal-zededa 's excellent suggestio to call it "cost" makes things a lot easier to explain.)